src/libraries/System.Reflection.MetadataLoadContext/src/System/Reflection/Runtime/BindingFlagSupport/PropertyPolicies.cs

```diff
@@ -82,12 +83,13 @@ public sealed override bool OkToIgnoreAmbiguity(PropertyInfo m1, PropertyInfo m2
             return false;
         }
 
-        private MethodInfo? GetAccessorMethod(PropertyInfo property)
+        private MethodInfo GetAccessorMethod(PropertyInfo property)
         {
             MethodInfo? accessor = property.GetMethod;
             if (accessor == null)
             {
                 accessor = property.SetMethod;
+                Debug.Assert(accessor != null);
```

Comment:
@buyaa-n note that requiring `ImplicitlyOverrides` take non-nullable arguments also forces we declare this method non-nullable too. Confident that this holds.